quarkus-build-caching-extension/pom.xml

```diff
@@ -143,6 +143,15 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
+            <resource>
+                <targetPath>META-INF</targetPath>
+                <directory>${basedir}</directory>
```

Comment:
Can you make it `project.basedir` here? We do that in other places in this POM, e.g., `project.build.directory`, and that's how I did it in CCUD Maven.

Reply:
Sure thing